.env.ci

```diff
@@ -21,6 +21,7 @@ DID_REGISTRY_ADDRESS=0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af
 CLAIM_MANAGER_ADDRESS=0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210
 DOMAIN_NOTIFIER_ADDRESS=0xeea658026d6CDede4380D3aD030beAC911758A93
 RESOLVER_V1_ADDRESS=0xf5EA22c1F799d425356c2aab2004200Ab4490D2b
+STAKING_POOL_ADDRESS=0x4b2A127680320eD980beAa7aD9b2447B96BC32fC
```

Comment:
It is better to name it `STAKING_POOL_FACTORY_ADDRESS` to not confuse with staking pool